pkg/pb/piecestore.proto

```diff
@@ -58,14 +58,14 @@ message PieceStore {
 
   RenterBandwidthAllocation bandwidth_allocation = 1;
   PieceData piece_data = 2;
-  SignedMessage authorization = 3;
+  PayerBandwidthAllocation payer_allocation = 3;
```

Comment:
`PayerBandwidthAllocation` should be inside `RenterBandwidthAllocation`. Most probably we don't need new field.

Reply:
I know we talked already but in case anyone else is reviewing--
the Put function on the piecestore client side only has PBA (not RBA) as a parameter. It also doesn’t even put RBA on the &pb.PieceStore message to be sent. So when you call `rba := recv.GetRBA()` on the server side it’s going to be nil. This can be verified currently on master. https://github.com/storj/storj/blob/39a1f175c4ea76ff3dc837a88d582e10d9bd6fb1/pkg/piecestore/psclient/client.go#L120